nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1348,13 +1350,15 @@ private Description handleInvocation(
             "passing @Nullable parameter '"
                 + state.getSourceForNode(actual)
                 + "' where @NonNull is required";
-        return errorBuilder.createErrorDescriptionForNullAssignment(
-            new ErrorMessage(MessageTypes.PASS_NULLABLE, message),
-            actual,
-            buildDescription(actual),
-            state);
+        passNullableDescriptionSet.add(
```

Comment:
Why not just call `state.reportMatch()` in each loop iteration, rather than collecting the descriptions in a set?  Not a huge difference, but the code would be slightly simpler if you eliminated the set